src/deterministic_vi/maximize_elbo.jl

```diff
@@ -80,12 +81,16 @@ function maximize_f{F}(f::F, ea::ElboArgs, transform::DataTransform;
         return hess
     end
 
-    tr_method = Optim.NewtonTrustRegion(
-                    initial_delta=10.0,
-                    delta_hat=1e9,
-                    eta=0.1,
-                    rho_lower=0.25,
-                    rho_upper=0.75)
+    if use_default_optim_params
+        tr_method = Optim.NewtonTrustRegion()
+    else
+        tr_method = Optim.NewtonTrustRegion(
```

Comment:
Can we just always use the default parameters rather than than having a flag that toggles between these settings? (Does something break with one setting but not the other?)